src/main/java/practice/StreamPractice.java

```diff
@@ -13,17 +17,29 @@
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        String messageNoSuchElExc = "Can't get min value from list";
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(",")))
+                .map(Integer::parseInt)
+                .filter(x -> x % 2 == 0)
+                .min(Integer::compare)
+                .orElseThrow(() -> new NoSuchElementException(messageNoSuchElExc));
```

Comment:
```suggestion
        return numbers.stream()
                .flatMap(n -> Arrays.stream(n.split(",")))
                .map(Integer::parseInt)
                .filter(x -> x % 2 == 0)
                .min(Integer::compare)
                .orElseThrow(() -> new NoSuchElementException("Can't get min value from numbers list " + numbers));
```

Обычно текст ошибки не выносят в отдельную переменную, так тяжелее читать код

Reply:
Зрозумів і виправив в IDEA, дякую